selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/onlyOffice/OnlyOfficeLocator.java

```diff
@@ -1,11 +1,25 @@
 package org.exoplatform.platform.qa.ui.selenium.locator.onlyOffice;
 
 import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.By;
 import static com.codeborne.selenide.Selenide.$;
 
 public class OnlyOfficeLocator {
 
     public static final SelenideElement ELEMENT_EDIT_ONlINE_BUTTON = $(".uiIconEcmsOnlyofficeOpen.uiIconEcmsLightGray.uiIconEdit");
     public static final SelenideElement ELEMENT_EDIT_ONLINE_BUTTON_FROM_PREVIEW = $(".onlyOfficeEditBtn");
+    public static final SelenideElement ELEMENT_DOWNLOAD_BUTTON_FROM_AS_TEXT = $(By.xpath("((//*[@class='pull-left statusAction'])[1]//a)[1]"));
+    public static final SelenideElement ELEMENT_OPEN_BUTTON_FROM_AS_TEXT = $(By.xpath("((//*[@class='pull-left statusAction'])[1]//a)[2]"));
+    public static final SelenideElement ELEMENT_EDIT_ONLINE_BUTTON_FROM_AS_TEXT = $(By.xpath("((//*[@class='pull-left statusAction'])[1]//a)[3]"));
```

Comment:
Instead of using xpath we should add a class on the button so you can select the element in a more robust way.

Reply:
I waiting for this comment :), just for this case I Used Xpath because I just need the alignment of elements of "pull left statusAction" and I compared with the next elements, also for download button I cant use ccs selector easily.

Reply:
> I waiting for this comment :)

:smile: 

> just for this case I Used Xpath because I just need the alignment of elements of "pull left statusAction" and I compared with the next elements, also for download button I cant use ccs selector easily.

The XPath is really fragile. If we add a button at the beginning for example, the test fails, even if OnlyOffice still work correctly. That's why we should only check Edit Online existence and add a class name on the element in the OnlyOffice code to select it easily.

Reply:
But the existing of Edit Online is checked in US1 and US2, This US is about alignment and I checked also with Sameh. it's about alignment.